heapster-saw/examples/iter_linked_list.sawcore

```diff
@@ -3,6 +3,22 @@ module iter_linked_list where
 
 import SpecM;
 
+-- Type description for the Mbox type, which is equivalent to Mbox_Ind
```

Comment:
Should this say `List`/`List_Ind` instead of `Mbox`/`Mbox_Ind`?

Reply:
Actually, the comment needed to be slightly different, but I fixed it, thanks!